sketch/tensor_full.hpp

```diff
@@ -0,0 +1,101 @@
+#pragma once
+
+#include "immintrin.h" // for AVX
+#include "nmmintrin.h" // for SSE4.2
+#include "sequence/alphabets.hpp"
+#include "sketch//sketch_base.hpp"
+#include "util/multivec.hpp"
+#include "util/timer.hpp"
+#include "util/utils.hpp"
+
+#include <cassert>
+#include <cmath>
+#include <iostream>
+#include <random>
+
+namespace ts { // ts = Tensor Sketch
+
+/**
+ * Computes tensor sketches for a given sequence as described in
+ * https://www.biorxiv.org/content/10.1101/2020.11.13.381814v1
+ * @tparam seq_type the type of elements in the sequences to be sketched.
+ */
+template <class seq_type>
+class FullTensor : public SketchBase<std::vector<double>, false> {
+  public:
+    // Tensor sketch output should be transformed if the command line flag is set.
```

Comment:
I don't understand the comment. Transformed into what? Which command line flag? 

Could you clarify the comment?

Reply:
deleted this - it's not relevant for this class.